### DISCUSSION TITLE
How to get a link for a header

### DISCUSSION THREAD

**Discussion by ghost** [squidfunk/mkdocs-material#7472] [Q&A]
How do I get the little icon next to a heading that says permanent link? Ive refered to Tooltips in the reference section and installed 
markdown_extensions:

  - abbr
  - attr_list
  - pymdownx.snippets
 
 as well as

theme:
  features:
    - content.tooltips
    
I added this and viewed in on the locahost 8000 website, but the link still didnt eppear. If im doing anything wrong, what should I do? I thought adding these would just automatically generate the permanent link. And if someone does answer this question, how do i change the icon of the permanent link? Any and all help is appreciated, thanks a lot.

**Comment by alexvoss** [comment]
I think what you are looking for is the configuration of the [`permalink` option in the Table of Contents](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#table-of-contents). You can set `permalink: true` to get the default symbol or provide your own symbol - just needs to be a unicode character.

**Reply by kamilkrzyskow** [reply]
The `permalink` setting only allows for unicode characters. If they want to use a special SVG icon they need to use CSS.
An example can be seen on the [PyMdown](https://facelessuser.github.io/pymdown-extensions/#pymdown-extensions) docs.
1. Set permalink to empty string:
- https://github.com/facelessuser/pymdown-extensions/blob/cd7c704487a3a79b6619bfcd0c6af83104d630a8/mkdocs.yml#L96
2. Then add extra CSS adapting the solution from there:
- https://squidfunk.github.io/mkdocs-material/customization/#additional-css
![image](https://github.com/user-attachments/assets/b3d9bfdd-fed0-4b8c-bdb3-ef7fa5fbc2c9)

EDIT:
Over at the Gothic Modding Community we did the same thing:
- https://github.com/Gothic-Modding-Community/gmc/blob/674ee2bac41a0d71b15ffc7744b66e928c1bdd22/overrides/assets/stylesheets/extra.css#L264-L282

**Reply by ghost** [reply]
Alright thanks. Will try the second one but if it's too complicated I'll just use permalink: true 😅thanks for the help though
